doc/introduction/connect.rst

```diff
@@ -0,0 +1,76 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _introduction/connect:
+
+*******
+Connect
+*******
+
+:term:`Connect` is the ftrack desktop GUI application enabling DCC application launch
+and publishing. Connect locates the framework plugins, and together with the
+:term:`Application launcher` plugin, enables discover and launch of DCC applications.
+
+The framework modules are connect plugins, which means that need to implement a launch
+hook containing the logic to discover and launch the DCC application.
+
+
+Application launcher
```

Comment:
@henriknorin-ftrack there's no need here to have reference to the configs, we have the applicatino launcher documentation.